examples/cookbook/jotai/TodoList.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { FlatList, Pressable, Text, View } from 'react-native';
+import { atom, createStore, useAtom } from 'jotai';
+
+export type TodoItem = {
+  id: string;
+  text: string;
+};
+
+export const todosAtom = atom<TodoItem[]>([]);
+
+export function TodoList() {
+  const [todos, setTodos] = useAtom(todosAtom);
+
+  const handleAddTodo = () =>
+    setTodos((prev) => [
+      ...prev,
+      {
+        id: Math.random().toString(36).slice(2, 11),
+        text: 'Buy almond milk',
+      },
+    ]);
+
+  if (!todos.length) {
+    return <Text>No todos, start by adding one...</Text>;
+  }
+
+  return (
+    <View>
+      <FlatList
+        data={todos}
+        renderItem={({ item }: { item: TodoItem }) => (
+          <Text key={item.id} accessibilityLabel={'todo-item'}>
```

Comment:
I would use `testID` instead of `accessiblityLabel` here, as this is for testing purposes and adding adding a11y label would mess with screen reader in normal apps